<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheFinalWindow</class>
 <widget class="QMainWindow" name="TheFinalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сапер от Rofin Incorporated</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow &gt; QWidget {
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_finalimage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>177</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_state">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Победа!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_level">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>На уровне  сложности 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rezult">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Бомб обезврежено: 20/20</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_NewGame">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Новая игра</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Exit">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Выйти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
